WinUIGallery.UITests/Properties/PublishProfiles/win-arm64ec.pubxml

```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--
+https://go.microsoft.com/fwlink/?LinkID=208121.
+-->
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <PublishProtocol>FileSystem</PublishProtocol>
+    <Platform>x64</Platform>
```

Comment:
Why should ARM64-ec point to x64 and not ARM?

Reply:
arm64ec and x64 are interchangeable. Arm64ec is basically x64 under the hood but with arm64

We were also running into this error that setting the pubxml to point to x64 runtime identifier solves. 
C:\ProgramFiles\dotnet\sdk\9.0.200\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(107,5): error NETSDK1083: The specified RuntimeIdentifier 'win-arm64ec' is not recognized. See https://aka.ms/netsdk1083 for more information. [C:\__w\1\s\Samples\WinUIGallery\WinUIGallery.UnitTests\WinUIGallery.UnitTests.csproj]